portfolio/src/main/webapp/script.js

```diff
@@ -21,7 +21,6 @@ let i = 0;
 
 setInterval(changePositionDisplayed, 3000);
 
-
 function  changePositionDisplayed(){
 	$('#position').fadeTo(300, 0).fadeTo(300, 1); 
```

Comment:
Looks like your indentation is off here.

Reply:
okay I will fix it